setup.py

```diff
@@ -16,14 +16,16 @@
         "six==1.10.0"
     ],
     install_requires=[
```

Comment:
Alphabetizing dependencies, removing google-auth and google-api-python-client, and adding six